syntax = "proto3";

import "map_renderer.proto";
import "transport_router.proto";

package serialize;

message Distance {
    string stop_from = 1;
    string stop_to = 2;
    int32 Distance = 3;
}
message Stop {
    string name = 1;
    repeated double coordinate = 2;

}

message Bus {
    string name = 1;
    repeated string stop = 2;
    bool is_circle = 3;
}

message TransportCatalogue {
    repeated Stop stop = 1;
    repeated Bus bus = 2;
    repeated Distance distance = 3;
    RenderSettings render_settings = 4;
    Router router = 5;
}